### PR TITLE
[WIP] [FR] Glossary translation

### PR DESCRIPTION
**Pull request by vincent38**
French translation of the glossary, done from line 1 to line 234
Still ordered by english words (will be reorganized when I finish translation)
All reviews and edits appreciated :)

### CODE REVIEW

File: wiki/Glossary/fr.md [hunk 1]
```diff
@@ -0,0 +1,999 @@
+# Glossaire
+
+Pendant des années, la osu!community a adopté son propre verlan qui peut être déroutant pour les débutants.
+Cette page est une compilation de la plupart de ces termes.
+Les ajouts et modifications sont les bienvenus !
+
+<!--Les liens préfixés par le symbole "#" sont aussi des termes de ce glossaire, tandis que les liens bleus pointent vers d'autres articles.-->
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: this no longer applies, you can remove it

File: wiki/Glossary/fr.md [hunk 2]
```diff
@@ -0,0 +1,999 @@
...
+
+## A
+
+### Friche abandonnée (terme original : Abandoned wasteland)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: if you are going to include the original term, please put it underneath instead

```
### Friche abandonnée

_Terme original : Abandoned wasteland_
```

File: wiki/Glossary/fr.md [hunk 3]
```diff
@@ -0,0 +1,999 @@
...
+Par exemple quand un utilisateur part faire quelque chose d'autre, alors qu'il est encore connecté.
+osu! désigne les joueurs inactifs depuis un certain temps avec la mention "Idle".
+
+### Cercle d'approche
+
+Dans le mode de jeu [osu!standard](/wiki/Game_Modes/osu!/), le cercle d'approche est le cercle extérieur qui entoure le cercle de frappe, le debut d'un slider, ou le centre d'un spinner.
+Quand le cercle d'approche touche la bordure extèrieure du cercle de frappe ou du spinner, vous devez cliquer.
+Pour les spinners, le cercle d'approche indiquera la durée de ce spinner.
+
+### Authentique (terme original : Authentic)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Authentique

_Terme original : Authentic_
```

File: wiki/Glossary/fr.md [hunk 4]
```diff
@@ -0,0 +1,999 @@
...
+Il supplante le besoin d'un serveur IRC tiers pour le management du contenu online d'osu!.
+Les joueurs se connectent automatiquement à Bancho quand ils démarrent osu!, et ce si ils sont connectés à internet
+
+### BAT
+
+**Au 1° février 2015, le BAT a été renommé le Beatmap Nomination Group**
+
+BAT&mdash;acronyme pour Beatmap Appreciation Team&mdash;était un groupe d'utilisateurs qui se chargeaient d'approuver et de ranker vos beatmaps .
+
+### Pulsation (terme original : Beat)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Pulsation

_Terme original : Beat_
```

File: wiki/Glossary/fr.md [hunk 5]
```diff
@@ -0,0 +1,999 @@
...
+La pulsation est une période de temps en lesquelles les chansons peuvent être divisées en fonction de leur BPM.
+Une beatmap bien timée aura toutes les pulsations qui correspondront au rythme de la musique, et une pulsation entière devra tomber entre deux lignes blanches sur la timeline.
+Dépendamment de la division appliquée, d'autres lignes de division de la pulsation peuvent être visibles:
+
+- lignes rouges pour les demi-pulsations,
+- lignes bleues pour les quarts de pulsations,
+- lignes jaunes pour les huitièmes de pulsations, et
+- lignes violettes pour les tiers et sixièmes de pulsations
+
+### Packs de beatmap (terme original : beatmap pack)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Packs de beatmap

_Terme original : beatmap pack_
```

File: wiki/Glossary/fr.md [hunk 6]
```diff
@@ -0,0 +1,999 @@
...
+
+Les Beatmap packs _(si le nom est ambïgu pour vous)_ sont des packs de beatmap.
+Les packs de beatmap sont des ensembles de beatmaps rankées et / ou approuvées qui peuvent être téléchargés depuis un service de partage de fichiers (comme [osu!Mirror](https://osu.yas-online.net/) ou [MediaFire](https://www.mediafire.com/)).
+
+La grande majorité des packs sont des ensembles de beatmaps rankées durant la même ppériode de temps. Cependant, certains peuvent contenir des chansons de thème ou de style similaires.
+
+Pour voir la liste des packs, consultez le [Listing des packs de beatmaps](https://osu.ppy.sh/p/packlist)
+
+### Espacement des pulsations (terme original : Beat spacing)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Espacement des pulsations

_Terme original : Beat spacing_
```

File: wiki/Glossary/fr.md [hunk 7]
```diff
@@ -0,0 +1,999 @@
...
+BN est l'acronyme pour Beatmap Nominator qui est un groupe d'utilisateurs chargés de nominer les beatmaps dites "en attente" (pending beatmaps).
+Ce groupe était précédemment connu sous l'acronyme de BAT (Beatmap Appreciation Team).
+
+Pour plus de détails à propos des Beatmap Nominators, consultez cet article : [Beatmap Nominator](/wiki/People/Beatmap_Nomination_Group/).
+
+### BPM
+
+BPM est l'acronyme de Beats Per Minute. Il représente le nombre de pulsations entières en une minute.
+
+### Pause (terme original : Break)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Pause

_Terme original : Break_
```

File: wiki/Glossary/fr.md [hunk 8]
```diff
@@ -0,0 +1,999 @@
...
+
+### Deathstreams
+
+Un deathstream est un long flux ou les objets se suivent les uns les autres en se succédant de manière rapide et/ou en formant des patterns assez confus.
+Ils amènent généralement le joueur à perdre sa "stamina" de manière rapide, et potentiellement à échouer la beatmap.
+
+Il est fortement suggéré d'utiliser les deathstreams uniquement lorsque la beatmap l'insinue et/ou qu'elle a un BPM élevé.
+Ce principe **ne devrait pas** être utilisé pour faire des overmaps!
+
+### Etoiles (terme original : Difficulty stars)
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Etoiles

_Terme original : Difficulty stars_
```

File: wiki/Glossary/fr.md [hunk 9]
```diff
@@ -0,0 +1,999 @@
...
+Les étoiles de difficulté sont un moyen de jauger la version de difficulté d'un niveau.
+Le "osu!website" trie les difficultés en 5 catégories (en anglais):
+
+- Easy,
+- Normal,
+- Hard,
+- Insane, and
+- Expert
+
+### Temps de drainage (terme original : drain time) <!--Review needed | max priority -->
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Temps de drainage

_Terme original : drain time_
```

File: wiki/Glossary/fr.md [hunk 10]
```diff
@@ -0,0 +1,999 @@
...
+## E
+
+### EBA
+
+EBA a deux sens:
+
+- Elite Beat Agents, un des jeux DS sur lesquels osu! est basé
+- un terme collectif désignant les personnages jouables dans le jeu ci-dessus, par exemple : "The EBA" ou "The Agents"
+
+### Chat étandu (terme original : extanded chat) -> https://github.com/ppy/osu-wiki/pull/100
```
Comment (MegaApplePi) [inline wiki/Glossary/fr.md]: ditto

if you are going to include the original term, please put it underneath instead

```
### Chat étandu

_Terme original : extanded chat_
```

**Comment by Ephemeralis** [comment]
Stale PR, no changes applied in over two months. Closing - feel free to reopen with changes applied.

**Pull request closed without merging**